mangaki/mangaki/models.py

```diff
@@ -355,7 +388,8 @@ def __str__(self):
 
 
 class Tag(models.Model):
-    title = models.TextField(unique = True)
+    title = models.CharField(max_length=255)
+    anidb_tag_id = models.IntegerField(default=0, unique=True)
```

Comment:
Yes, you can use a one-off default which would be 0 for example, but we will perform such a migration on empty tables, it'd be a no-operation.
(maybe let's add a comment to say this is going to happen and we assume you run this on empty tags' tables).